src/cli/open_editor/open_editor.gd

```diff
@@ -15,19 +17,51 @@ func _init():
 
 func execute(req: Request) -> void:
 	if _editors.load() == Error.OK:
-		var editors = _editors.filter_by_name_pattern(req.name_pattern)
-		if editors.size() == 1:
-			_run_editor(editors[0], req.user_args)
-		elif editors.size() > 1:
-			var names: Array[String] = []
-			for e in editors:
-				names.append(e.name)
-			Output.push("There is ambiguity between editors to run.\n%s" % "\n".join(names))
+		var editor: LocalEditors.Item = null
+
+		if req.name_pattern.is_empty():
+			var project_path = DirAccess.open(req.working_dir).get_current_dir() + "/project.godot"
+			editor = _find_editor_by_proj_godot(project_path)
 		else:
-			Output.push("Required editor is not found!")
+			editor = _find_editor_by_name(req.name_pattern)
+
+		if editor != null:
+			_run_editor(editor, req.user_args)
 	else:
 		Output.push("Some error happend during editors loading!")
 
+func _find_editor_by_proj_godot(project_path: String) -> LocalEditors.Item:
+	var result: LocalEditors.Item = null
+	
+	if FileAccess.file_exists(project_path):
+		var project_info = Projects.ExternalProjectInfo.new(project_path)
```

Comment:
we should also check if a project with the given project_path exists, and if it does - use the editor bound to it